doc/lib/attributes.py

```diff
@@ -76,6 +77,8 @@ def attr_to_table_row(attr_spec, prefix=None):
                 # recurse to child element
                 if issubclass(type_, model.NamedModelElement):
                     child_element = type_(name='<user-chosen>', raw_dict={})
```

Comment:
as discussed yesterday: there should be no need to instantiate the elements. just use `type_` in all cases

Reply:
removed instantiation of elements